### PR TITLE
[Refactor] Riot is now Element + Molecule tests

### PR DESCRIPTION
**Pull request by santiagomr**
Now that [Riot is Element](https://element.io/blog/welcome-to-element/), this PR implements the refactoring required to keep the web client provisioning up to date.

Due to the loss of backward compatibility, these changes involve releasing `v2.0.0` of the role.

Additionally, a new Molecule test scenario was implemented to contemplate a [full installation](https://github.com/UdelaRInterior/ansible-role-matrix-synapse/tree/master#full-installation)

### CODE REVIEW

**Comment by vamgnu** [comment]
To generate an identical scenario was very useful add a variable to indicate the same version installed in prod state.
I made tests for update our matrix-synapse instances adding `matrix_synapse_pip_pkg` variable on [these line](https://github.com/UdelaRInterior/ansible-role-matrix-synapse/blob/89b34cb0fb224352ae030d738d4adf279907f8bb/tasks/install_from_source.yml#L41) . And set `matrix-synapse` as default value.

As usual, sorry about my english

**Comment by santiagomr** [comment]
Effectively, it's a very useful feature for the role to be able to indicate the version of synapse to install using a variable.

Do you plan to propose those changes in a new branch or right here?

**Comment by vamgnu** [comment]
How do you recommend doing it? I'm newbie on these... things

**Comment by vamgnu** [comment]
> Effectively, it's a very useful feature for the role to be able to indicate the version of synapse to install using a variable.
> 
> Do you plan to propose those changes in a new branch or right here?

As you mentioned by chat. I create a [new branch](https://github.com/UdelaRInterior/ansible-role-matrix-synapse/tree/riot-is-now-element-PR-3-issuecomment-712465673), made my changes and [pushed a commit with it](https://github.com/UdelaRInterior/ansible-role-matrix-synapse/commit/53052df74425ee708ccd91816930dc4890c7ac5a).
I'll be waiting how to proceed....

**Comment by santiagomr** [comment]
Great! You should open a new PR with source in the branch you created to propose the feature and destination [the branch you started from](https://github.com/UdelaRInterior/ansible-role-matrix-synapse/tree/riot-is-now-element). 

Once the PR is created, assign the revision to me.

**Comment by santiagomr** [comment]
I just fixed some details to make it work on the recently released Ansible 2.10 as well (and to pass the tests on Travis CI).

Can we take this refactoring to master so that everything is up to date in the stable version?

**Comment by vamgnu** [comment]
I didn't review the tests on Travis CI yet... Give me some time to do it.

File: templates/var/lib/matrix-synapse/conf.d/email.yaml.j2 [hunk 1]
```diff
@@ -20,11 +20,11 @@ email:
 #   #
 #   notif_for_new_users: true
 #
-#   # Defining a custom URL for Riot is only needed if email notifications
-#   # should contain links to a self-hosted installation of Riot; when set
+#   # Defining a custom URL for Element is only needed if email notifications
+#   # should contain links to a self-hosted installation of Eiot; when set
```
Comment (vamgnu) [inline templates/var/lib/matrix-synapse/conf.d/email.yaml.j2]: Seems like a typing error. Should be "Element" instead of "Eiot"?

**Review by vamgnu — CHANGES_REQUESTED** [review]
Pass for "inline review"

File: templates/var/lib/matrix-synapse/conf.d/email.yaml.j2 (hunk 1 above)
Comment (santiagomr) [reply templates/var/lib/matrix-synapse/conf.d/email.yaml.j2]: That's right! That was half done

**Review by vamgnu — APPROVED** [review]
The changes was tested on real scenario to upgrade our matrix-synapse instance. Was the best review.
To not waste any more time, i propose merge these changes to master branch. Pending to upgrade molecule tests.

**Pull request merged**